kubeflow/core/centraldashboard.libsonnet

```diff
@@ -23,52 +23,25 @@
         namespace: namespace,
       },
       spec: {
-        replicas: 1,
-        selector: {
-          matchLabels: {
-            app: "centraldashboard",
-          },
-        },
         template: {
           metadata: {
-            creationTimestamp: null,
             labels: {
               app: "centraldashboard",
             },
           },
           spec: {
             containers: [
               {
-                env: [
-                  {
-                    name: "CENTRALDASHBOARD_NAMESPACE",
-                    valueFrom: {
-                      fieldRef: {
-                        apiVersion: "v1",
-                        fieldPath: "metadata.namespace",
-                      },
-                    },
-                  },
+                image: "swiftdiaries/centraldashboard:0.3",
```

Comment:
Is there any issue to start publishing this image some where? If so lets link to the issue here.

Reply:
https://github.com/kubeflow/kubeflow/issues/527

Reply:
I'll add a commit with the changes.

Reply:
Can you add a TODO and reference the GitHub issue please?

Reply:
Thank you the review.

Reply:
As soon as you add the TODO this should be ready to go.